spec/requests/pages_spec.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+describe "Pages", type: :request do
+  let(:user) { create(:user) }
+
+  describe "#home" do
+    it "respond to request" do
+      get pages_home_path
+      expect(response).to be_successful
+    end
+
+    context "logged in" do
+      before { sign_in user }
+
+      it "has no stories" do
+        list = double
+        expect(Kaminari).to receive(:paginate_array).and_return(list)
+        expect(list).to receive(:page)
+        get pages_home_path
+      end
+
+      it "has stories" do
+        create(:strategy, user_id: user.id)
+        categories = create_list(:category, 2, user_id: user.id)
+        moods = create_list(:mood, 2, user_id: user.id)
```

Comment:
I'd also like to add that these should be moved outside of the "it" block and into `let()` assignments

Reply:
Where should those `let( )` assignments happen? The `user` variable won't be available in 'describe' or 'context' blocks

Reply:
We could use `let!` but that differs from what @bennpham requested in their review from line 46

Reply:
I don't see it used anywhere else to put it in `let()` assignments looking at the test itself. I think we'd be over abstracting it at that point. If we had a context for maybe `when user have stories` and multiple `it` blocks, then I can see using `let!` here unless I'm missing something.